templates/ocs-deployment/filesystem.yaml

```diff
@@ -0,0 +1,54 @@
+#################################################################################################################
+# Create a filesystem with settings with replication enabled for a production environment.
+# A minimum of 3 OSDs on different nodes are required in this example.
+#  kubectl create -f filesystem.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: myfs
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+spec:
+  # The metadata pool spec. Must use replication.
+  metadataPool:
+    replicated:
+      size: 3
+  # The list of data pool specs. Can use replication or erasure coding.
+  dataPools:
+    - failureDomain: host
+      replicated:
+        size: 3
+  # The metadata service (mds) configuration
+  metadataServer:
+    # The number of active MDS instances
+    activeCount: 1
```

Comment:
Please make this as param, where default 1 MDS count, or the user can specify the count.

Reply:
+1